go.mod

```diff
@@ -2,14 +2,15 @@ module github.com/asecurityteam/serverfull
 
 go 1.22
 
-toolchain go1.23.1
+toolchain go1.23.3
 
 require (
 	github.com/asecurityteam/logevent/v2 v2.0.1
 	github.com/asecurityteam/runhttp v0.6.5
 	github.com/asecurityteam/settings/v2 v2.0.1
 	github.com/aws/aws-lambda-go v1.47.0
 	github.com/go-chi/chi v1.5.5
```

Comment:
Question - do we need the old chi around?

Reply:
Ugh probably not, but it's probably due do other dependencies and needs to be manually cleaned up... I won't bother with this PR for now then.